Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -109,6 +112,36 @@ MainWindow::MainWindow(std::unique_ptr<BootParameters> boot_parameters) : QMainW
 
   NetPlayInit();
 
+#if defined(__unix__) || defined(__unix) || defined(__APPLE__)
```

Comment:
This block should probably go in its own dedicated function instead of being dropped wholesale in the constructor.

Reply:
Done.